Remove oranda in favour of zola pages

tertsdiepraam opened the pull request:
Closes https://github.com/uutils/uutils.github.io/issues/33

Styling can be improved a lot but I like how this gives us much more control.

Also I changed the path for the book, because `docs` makes much more sense. Now it really looks like I want it to.

cakebaker commented on content/coreutils.md:
```diff
@@ -0,0 +1,23 @@
+uutils aims to be a drop-in replacement for the GNU utils. Differences with GNU are treated as bugs.
+
+uutils aims to work on as many platforms as possible, to be able to use the same utils on Linux, Mac, Windows and other platforms. This ensures, for example, that scripts can be easily transferred between platforms.
+Contributing
```
```suggestion

# Contributing
```

tertsdiepraam commented:
I guess I should add a bit of text for diffutils as well before we merge this.

tertsdiepraam commented:
And findutils should get some documentation 🤔

tertsdiepraam commented:
The failure will be resolved once https://github.com/uutils/findutils/pull/347 is merged. Then we can also do the same for `diffutils`.

tertsdiepraam commented:
I'll add a @BenWiederhake style "changes since last push":
- fleshed out the pages for the projects a bit with CONTRIBUTING.md (if available) and licenses.
- Fixed the link to the installation instructions.
- Note that diffutils still needs documentation, so its links will be broken until we add that.

tertsdiepraam commented:
And changed some wording in coreutils.md. :)

cakebaker commented on content/diffutils.md:
```diff
@@ -0,0 +1,16 @@
+
++++
+
+Rust implementation of GNU diffutils: `diff`, `cmp` and `diff`.
```
https://www.gnu.org/software/diffutils/ lists four utils:
```suggestion
Rust implementation of GNU diffutils: `diff`, `cmp`, `diff3` and `sdiff`.
```

tertsdiepraam commented:
Let's see how this fares :)
